fendermint/app/src/cmd/genesis.rs

```diff
@@ -45,6 +45,7 @@ cmd! {
       power_scale: self.power_scale,
       validators: Vec::new(),
       accounts: Vec::new(),
+      eam_permission_mode: PermissionMode::Unrestricted,
```

Comment:
How does the user specify the permission mode when constructing genesis?

Reply:
This is the default setup, so when a new genesis is created by calling fendermint, the default mode is `Unrestricted`. Then users can update this field to restricted.